staging/src/k8s.io/metrics/pkg/apis/metrics/types.go

```diff
@@ -85,5 +85,19 @@ type ContainerMetrics struct {
 	// Container name corresponding to the one from pod.spec.containers.
 	Name string
 	// The memory usage is the memory working set.
-	Usage api.ResourceList
+	Usage ResourceList
 }
+
+// NOTE: ResourceName and ResourceList are copied from
```

Comment:
@lavalamp I moved the copied types to the types.go and explained why the copies are needed. These copies are permanent. 

cc @DirectXMan12